src/app/locator/qgsinbuiltlocatorfilters.h

```diff
@@ -123,11 +141,15 @@ class APP_EXPORT QgsActiveLayerFeaturesLocatorFilter : public QgsLocatorFilter
     QString mLayerId;
     QIcon mLayerIcon;
     QStringList mAttributeAliases;
+    QStringList mFieldsCompletion;
     int mMaxTotalResults = 30;
 
     friend class TestQgsAppLocatorFilters;
 };
 
+Q_DECLARE_METATYPE( QgsActiveLayerFeaturesLocatorFilter::Data )
```

Comment:
Would be cleaner to avoid the Data struct and instead use a QVariantMap value.

Reply:
done